updatehub/tests/common.rs

```diff
@@ -49,18 +50,24 @@ impl Default for Settings {
             download_dir: None,
             config_file: None,
             timeout: None,
+            install_modes: None,
```

Comment:
Couldn't this to be a `Vec<_>`?

Reply:
We could, but then we won't be able to use the derived version of `Default` trait and would need a explicit impl for the struct.

Reply:
Oh, makes sense.